medic/medic-log.js

```diff
@@ -63,10 +63,14 @@ function logIOS() {
     });
 }
 
-function logWindows() {
+function logWindows(timeout) {
     var logScriptPath = path.join("mobilespec", "platforms", "windows", "cordova", "log.bat");
     if (fs.existsSync(logScriptPath)) {
-        shelljs.exec(logScriptPath + " --dump --mins 15", function (code, output) {
+        var mins = 15;
+        if (timeout) {
+            mins = Math.ceil(timeout / 60);
```

Comment:
A few notes:
- Please add a small constant (~2 min) to the time so that we also capture anything that happened before the run
- Please factor out the default 15 mins into a constant
- Please add a function to convert minutes to seconds, so it's easier to read (i.e. `mins = sec2min(timeout)`)